README-DEV.md

```diff
@@ -125,15 +125,15 @@ Start a local pool as described above, and then just execute
 dcld
 ```
 
-Have a look at [How To](docs/how-to.md) and [transactions](docs/transactions.md) for instructions how to configure and use the CLI.
+Have a look at [How To](docs/how-to.md) and [transactions](docs/transactions/transactions.md) for instructions how to configure and use the CLI.
```

Comment:
Can we please keep the main reference to original transactions.md unchanged? It's already shared in many places within community.

Reply:
Fixed